sw/device/tests/crypto/cryptotest/firmware/extclk_sca_fi.c

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/tests/crypto/cryptotest/firmware/extclk_sca_fi.h"
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/ottf_test_config.h"
+#include "sw/device/lib/testing/test_framework/ujson_ottf.h"
+#include "sw/device/lib/ujson/ujson.h"
+#include "sw/device/tests/crypto/cryptotest/json/extclk_sca_fi_commands.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+static dif_clkmgr_t clkmgr;
+
+status_t handle_extclk_sca_fi_configure(ujson_t *uj) {
+  cryptotest_extclk_sca_fi_cfg_t uj_data;
+  TRY(ujson_deserialize_cryptotest_extclk_sca_fi_cfg_t(uj, &uj_data));
+
+  TRY(dif_clkmgr_init(mmio_region_from_addr(TOP_EARLGREY_CLKMGR_AON_BASE_ADDR),
+                      &clkmgr));
+
+  bool is_low_speed = true;
```

Comment:
if you made the uJSON struct types a bool, I think you can just use the value throughout, rather than having to parse the UJSON struct and convert it to a bool

Reply:
applies to the external clock select field too